boris.martynov/task-1/internal/calculator/calculator.go

```diff
@@ -0,0 +1,56 @@
+package calc
+
+import (
+	"fmt"
+)
+
+func Add(a, b int64) int64 {
+	return a + b
+}
+
+func Subt(a, b int64) int64 {
+	return a - b
+}
+
+func Mul(a, b int64) int64 {
+	return a * b
+}
+
+func Div(a, b int64) (int64, bool) {
+	if b == 0 {
+		return 0, false
+	}
+	return a / b, true
+}
+
+func UserInput() (int64, int64, string) {
+	var a, b int64
+	var operator string
+	var n int
+	var err error
+
+	n, err = fmt.Scanln(&a)
```

Comment:
зачем плодить переменные выше, если просто можно
```suggestion
	n, err := fmt.Scanln(&a)
```